app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Admin::BaseController < ApplicationController
+
+  before_action :require_admin
+
+  def require_admin
+    render file: "/public/404" unless current_user && current_user.admin?
```

Comment:
@Dpalazzari @njgheorghita I was wondering the same thing here. I was thinking a 403 might be better or the root and forbidden message like @Laszlo-JFLMTCO  mentioned